feapi/app/setup.py

```diff
@@ -7,7 +7,7 @@
 
 inst_reqs = [
     "pydantic==1.8.2",
-    "pygeofilter",
+    "pygeofilter==0.0.2",
```

Comment:
It is not clear to me if `pygeofilter` is required here. An OS dependency is required to build with an Alpine docker base image so it is also included in `requirements_build.txt`.
`scripts/test` is successful both with and without pygeofilter here.